popup/popup.js

```diff
@@ -244,4 +291,12 @@ document.querySelectorAll('[translate-href]').forEach(function(element) {
   translateHref(element, element.getAttribute('translate-href'));
 });
 
+document.querySelectorAll('[translate-html]').forEach(function(element) {
+  translateHtml(element, element.getAttribute('translate-html'));
+});
+
+document.querySelectorAll('[title]').forEach(function(element) {
+  translateA11y(element, element.getAttribute('title'));
+});
```

Comment:
translateHtml must be called after translateA11y in order to not remove already translated title (for example kmByCarUnit)
You need to switch L294-296 after L300